cypress-browser-using-plugin/cypress.config.js

```diff
@@ -0,0 +1,13 @@
+const { defineConfig } = require("cypress");
+const registerSealights =
+  require("SL.Cypress.Plugin/dist/code-coverage/config").default;
```

Comment:
it is better not to use default import in public libraries or to make both options of import available
require("SL.Cypress.Plugin/dist/code-coverage/config") - for common js
and import "SL.Cypress.Plugin/dist/code-coverage/config" - for EsModules

I mean that export should looks like
module.exports = foo;
module.exports.default = foo;

and in esModules better to avoid it at all